src/main/webapp/WEB-INF/themes/desktop/xform/article-transform.xsl

```diff
@@ -206,7 +206,17 @@
         </xsl:for-each>
         <!-- pub year -->
         <xsl:text> (</xsl:text>
-        <xsl:value-of select="pub-date[@pub-type='collection']/year | pub-date[@pub-type='ppub']/year"/>
+        <xsl:choose>
+          <xsl:when test="pub-date[@pub-type='collection']/year | pub-date[@pub-type='ppub']/year">
+            <xsl:value-of select="pub-date[@pub-type='collection']/year | pub-date[@pub-type='ppub']/year"/>
+          </xsl:when>
+          <xsl:when test="pub-date[@pub-type='epub']/year">
+            <xsl:value-of select="pub-date[@pub-type='epub']/year"/>
+          </xsl:when>
+          <xsl:otherwise>
```

Comment:
this is a strange otherwise, I think we should test for it. and include nothing - no parenthesis when we have no "pubdate".